spec/support/html_spec_helper.rb

```diff
@@ -1,10 +1,10 @@
 module HTMLSpecHelper
-  def fake(uri, response, verb = :get)
-    FakeWeb.register_uri(verb, uri, response: header(response))
+  def fake(uri, response, status = 200, verb = :get)
```

Comment:
I modify the fake() method to support 404 error on SPOJ when the problem code is invalid. The default value of param status is 200(OK). You can change the implement if you want.